packages/next/src/server/dev/turbopack-utils.ts

```diff
@@ -118,7 +118,14 @@ export function formatIssue(issue: Issue) {
   }
 
   if (description) {
-    message += renderStyledStringToErrorAnsi(description) + '\n\n'
+    if (
+      description.type === 'text' &&
+      description.value.startsWith('Import map: No import map entry')
```

Comment:
Let's remove this message from turbopack instead, only showing it if there is a sourcemap

Reply:
Where do we emit this message? I could see where it comes from (https://github.com/vercel/turbo/blob/main/crates/turbopack-core/src/resolve/options.rs#L332) and some existing test have same (https://github.com/vercel/turbo/blob/97950cd84b7278b6b3b16ee9bc9d96c303ddd4d6/crates/turbopack-tests/tests/snapshot/imports/resolve_error_esm/issues/__l___Module%20not%20found____c__%20Can't%20resolve%20'__c_d-b93f4e.txt#L11) but unsure where it is being emitted.